### PR TITLE
Test

### PR DESCRIPTION
**Pull request by chaserileyroberts**

### CODE REVIEW

File: qsimcirq/qsim_simulator.py [hunk 1]
```diff
@@ -62,6 +64,130 @@ def __init__(self, qsim_options: dict = {}):
...
+      boundlen = bound[1]-bound[0]
+      results[key] = np.ndarray(shape=(repetitions, bound[1]-bound[0]), dtype=int)
+
+    for i in range(repetitions):
+      qsim_state = qsim.qsim_simulate_fullstate(options)
+      amplitudes = QSimSimulatorState(qsim_state, qubit_map).state_vector
+
+      # Convert amplitudes to probabilities
+      probabilities = np.array(
+        [(a*complex(a.real, -a.imag)).real for a in amplitudes]
```
Comment (chaserileyroberts) [inline qsimcirq/qsim_simulator.py]: I think you can just do `np.abs(amplitudes) ** 2`

File: qsimcirq/qsim_simulator.py [hunk 2]
```diff
@@ -62,6 +64,130 @@ def __init__(self, qsim_options: dict = {}):
...
+      )
+      probabilities /= probabilities.sum()
+
+      # format the bitstring in terms of measurement gates
+      result = np.random.choice(bitstrings, p=probabilities)
+      print(bounds)
+      print(result)
+      print(bitstrings)
+      print(probabilities)
```
Comment (chaserileyroberts) [inline qsimcirq/qsim_simulator.py]: Remove print statements.

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Runtime requirements for the python 3 version of cirq.
 
-cirq==0.8.0
+cirq==0.9.0.dev0
```
Comment (chaserileyroberts) [inline requirements.txt]: Requiring dev version might be bad. What do you need from it?

**Review by chaserileyroberts — COMMENTED** [review]
Looks good, small nits.

File: requirements.txt (hunk 3 above)
Comment (TylerLeite) [reply requirements.txt]: cirq.sim.WaveFunctionSimulatorState and cirq.sim.WaveFunctionTrialResult are deprecated and replaced by cirq.sim.StateVectorSimulatorState and cirq.sim.StateVectorTrialResult, but those don't exist in 0.8.x. Master branch of the cirq library has these changes. It's not 100% necessary for these changes, but they do use the SimulatorState, and therefore will be part of the upgrade-refactor that will eventually be needed anyway.

**Pull request closed without merging**